templates/default/dockerImage.sh.erb

```diff
@@ -99,6 +99,13 @@ elif [ "$1" == "export" ] ; then
     docker pull $2 > /dev/null 2>&1
     docker run --rm $2 conda env export -n theenv
 
+elif [ "$1" == "loginaz" ] ; then
```

Comment:
Same comment as the other PR
https://github.com/logicalclocks/hopsworks-chef/pull/589#discussion_r599740069